hoomd/logging.py

```diff
@@ -595,9 +595,8 @@ class they come from. For instance, the `hoomd.md.pair.LJ` class has a
             the only types of loggable quantities that can be logged by this
             logger. Defaults to allowing every type.
         only_default (`bool`, optional): Whether to log only quantities that are
-            logged by "default", defaults to ``True``. This mostly means that
-            performance centric loggable quantities will be passed over when
-            logging when false.
+            logged by "default". Defaults to ``True``. If ``False``, loggable
```

Comment:
Why the quotes on `by "default"`?
Where is this "default" specified?

Reply:
I think emphasizing it is confusing because the parameter also contains "default". Would we lose much by omitting this bit?